src/go/rpk/pkg/api/admin/api_cloud_storage_test.go

```diff
@@ -0,0 +1,205 @@
+// Copyright 2023 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package admin
+
+import (
+	"context"
+	"encoding/json"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type (
+	testRequestHandler func(w http.ResponseWriter, r *http.Request)
+	testFn             func(t *testing.T) testRequestHandler
+	requestTestCase    struct {
+		name             string
+		testFn           testFn
+		shouldError      bool
```

Comment:
We can remove `shouldError` since all are false in the below test cases.

Reply:
Good catch. I removed a bunch of test cases that are already covered by `admin_test.go` and didn't notice that included all the failure cases

I'll remove it

Reply:
Does this change work for you as a fix https://github.com/redpanda-data/redpanda/pull/8836/commits/ae89dd9cd57011fcd340fc1a2fbd339bdec18733?

Reply:
Yes, thanks